extensions/Khronos/KHR_materials_pbrSpecularGlossiness/README.md

```diff
@@ -0,0 +1,214 @@
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Patrick Cozzi, Cesium, [@pjcozzi](https://twitter.com/pjcozzi)
+
+## Status
+
+Draft 
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines the specular-glossiness material model from Physically-Based Rendering (PBR). This extensions allows glTF to support this additional workflow. 
+
+The [conformance](#conformance) section specifies what an implementation must to do when encountering this extension, and how the extension interacts with the materials defined in the base specification.
+
+## Extending Materials
+
+The PBR specular-glossiness materials are defined by adding the `KHR_materials_pbrSpecularGlossiness` extension to any glTF material. 
+For example, the following defines a material like gold using specular-glossiness parameters.:
+
+```
+{
+    "materials": [
+        {
+            "name": "gold",
+            "extensions": {
+                "KHR_materials_pbrSpecularGlossiness": {
+                    "diffuseFactor": [ 0.5, 0.5, 0.5, 1 ],
+                    "specularFactor": [ 0.0, 0.0, 0.0 ],
+                    "glossinessFactor": 0.8
+                }
+            }
+
+        }
+    ]
+}
+```
+
+### Specular - Glossiness
+
+Todo: Add figure
+
+The specular-glossiness material model is defined by the following properties:
+* `diffuse` - Reflected diffuse color of the material
+* `specular` - Specular color of the material
+* `glossiness` - Glossiness of the material
+
+The diffuse value represents the reflected diffuse color of the material. The specular value defines the specific measured reflectance value at normal incidence (F0). The glossiness property is a factor between `0.0` (rough surface) and `1.0` (smooth surface) and represents the surface irregularities that cause light diffusion. 
+
+The specular property from specular-glossiness material model is the same as the base color value from the metallic-roughness material model for metals. The glossiness property from specular-glossiness material model is related with the roughness property from the metallic-roughness material model and is defined as `glossiness = 1 - roughness`. See [appendix](#appendix) for more details on how you can convert between these two material models.
+
+The value for each property (`diffuse`, `specular`, `glossiness`) can be defined using factors or textures. The `specular` and `glossiness` properties are packed together in a single texture called `specularGlossinessTexture`.  If a texture is not given, all respective texture components within this material model are assumed to have a value of `1.0`. The factors (`diffuseFactor`, `specularFactor`, `glossinessFactor`) scale, in linear space, the components given in the respective textures (`diffuseTexture`, `specularGlossinessTexture`). Texture content must be converted to linear space before it is used for any lighting computations.  
```

Comment:
> Texture content must be converted to linear space

Does it imply that all textures should be stored in sRGB? If so, we must mention that.
Should it be possible to directly store linear data (e.g., via floats in future) in textures? Also, see #835.